mongodbatlas/resource_mongodbatlas_cloud_backup_snapshot_restore_job.go

```diff
@@ -320,6 +320,11 @@ func resourceMongoDBAtlasCloudBackupSnapshotRestoreJobDelete(ctx context.Context
 		shouldDelete = false
 	}
 
+	if aut, ok := d.Get("delivery_type_config.0.point_in_time").(bool); ok && aut {
+		log.Print("Point in time restore cannot be cancelled")
+		shouldDelete = false
+	}
```

Comment:
@andreaangiolillo can you also update our docs accordingly letting users know that this behavior is not allowed: https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/cloud_backup_snapshot_restore_job#point-in-time

If possible can we also link back to Atlas source docs if possible that highlights this restriction in Point in Time restore jobs?

Reply:
Updated.

> If possible can we also link back to Atlas source docs if possible that highlights this restriction in Point in Time restore jobs?

I haven't found a doc link that highlights this limitation

Reply:
SGTM